engine/apps/alerts/migrations/0045_alertreceivechannel_mobile_app_message_template_and_more.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.7 on 2024-02-06 20:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('alerts', '0044_alertreceivechannel_alertmanager_v2_backup_templates_and_more'),
+    ]
+
+    operations = [
```

Comment:
I think we can avoid the additional columns by using the TemplateLoader and store in  messaging_backends_templates instead. See next comment